aries_cloudagent/protocols/present_proof/v2_0/routes.py

```diff
@@ -322,7 +305,14 @@ async def _add_nonce(indy_proof_request: Mapping) -> Mapping:
 
 def _formats_attach(by_format: Mapping, msg_type: str, spec: str) -> Mapping:
     """Break out formats and proposals/requests/presentations for v2.0 messages."""
-
+    attach = []
+    for (fmt_api, item_by_fmt) in by_format.items():
+        if fmt_api == V20PresFormat.Format.INDY.api:
+            attach.append(
+                AttachDecorator.data_base64(mapping=item_by_fmt, ident=fmt_api)
+            )
+        elif fmt_api == V20PresFormat.Format.DIF.api:
+            attach.append(AttachDecorator.data_json(mapping=item_by_fmt, ident=fmt_api))
```

Comment:
I don't think we need to make a distinction between the formats for the data format. I'd just go with base64 for both

Reply:
I went by the RFC examples which had JSON format, but I can change this.
https://github.com/hyperledger/aries-rfcs/blob/master/features/0510-dif-pres-exch-attach/README.md#propose-presentation-attachment-format

Reply:
For now going with json data format for DIF PresExchange